RebelAllianceBank/Interfaces/IBankAccount.cs

```diff
@@ -2,6 +2,8 @@
 {
     public interface IBankAccount
     {
+        public int ID { get; set; }
+        public int AccountType { get; set; }
```

Comment:
Why is it int and not string?

Reply:
If you meant the UserId, it's because of it's a reference to the IUser PersonalNum